pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -614,8 +616,8 @@ def main():
     arg_dict = gd.cli("commuter_official")
     ref_year = 2022
 
-    abs_tol = 100  # maximum absolute error allowed per county migration
-    rel_tol = 0.01  # maximum relative error allowed per county migration
+    abs_tol = 100  # maximum absolute error allowed per county mobility
+    rel_tol = 0.01  # maximum relative error allowed per county mobility
```

Comment:
Did you make a test run that it returns the same matrix as the one stored in the data folder?

Reply:
Yes, the matrix is the same.